src/cfn.py

```diff
@@ -41,11 +44,31 @@ def ensure_destroyed(stackname):
 @task(alias='aws_update_stack')
 @requires_aws_stack
 @timeit
-def update(stackname):
+def update(stackname, part_filter=None):
```

Comment:
`part_filter` means nothing to me. I can't even deduce it's purpose looking at the `buildercore` code. This parameter needs a more obvious name and the docstring should be updated